spec/controllers/concerns/recaptcha_concern_spec.rb

```diff
@@ -20,4 +20,52 @@ def index
       expect(csp.frame_src).to include(*RecaptchaConcern::RECAPTCHA_FRAME_SRC)
     end
   end
+
+  describe '#add_recaptcha_resource_hints' do
+    controller ApplicationController do
+      include RecaptchaConcern
+
+      after_action :add_recaptcha_resource_hints
+
+      def index
+        if params[:add_link]
+          response.headers['Link'] = '<https://example.com>;rel=preconnect'
+        end
+
+        render plain: ''
+      end
+    end
+
+    subject(:response) { get :index }
+    let(:processed_links) do
+      response.headers['Link'].split(',').map { |link| link.chomp.split(';').map(&:chomp) }
```

Comment:
With the `.map(&:chomp)` the intermediate `.chomp` is not strictly necessary
```suggestion
      response.headers['Link'].split(',').map { |link| link.split(';').map(&:chomp) }
```

Reply:
Ah yeah I suppose you're right about that! Good catch.